common/src/main/java/com/wynntils/wynn/item/parsers/WynnItemMatchers.java

```diff
@@ -40,7 +42,8 @@ public final class WynnItemMatchers {
             Pattern.compile(".*?§([2ebcf])([✤✦❉✹❋]) (Strength|Dexterity|Intelligence|Defence|Agility).*?");
     private static final Pattern TELEPORT_SCROLL_PATTERN = Pattern.compile(".*§b(.*) Teleport Scroll");
     private static final Pattern TELEPORT_LOCATION_PATTERN = Pattern.compile("- Teleports to: (.*)");
-    private static final Pattern DUNGEON_KEY_PATTERN = Pattern.compile("(?:§.)*(?:Broken )?(?:Corrupted )?(.+) Key");
+    private static final Pattern DUNGEON_KEY_PATTERN =
+            Pattern.compile("(?:§.)*(?:\\d*x )*(?:§.)*(?:Broken )?(?:Corrupted )?(.+) Key");
```

Comment:
This doesn't fix the whole issue. In fact this is a really patchy fix that only fixes a single regex, other regexes are broken.

Reply:
oh right the other items have that issue too, I'll fix those too

Reply:
No, don't do that. We need to handle them internally, somewhere else. @magicus could share his opinion on this

Reply:
@kristofbolyai Not entirely sure what you mean. Like when in the trade market, we should prepend a number prefix to the regexp?

Reply:
Maybe. I am not sure how to deal with this.

Reply:
Maybe we need like a "InventoryTypeItemNameRewriter" which, when in the TradeMarket, removes the count ... eh... no, sounds bad as well. I don't know either. 

Does it affect other types of items apart from dungeon keys? Otherwise, if this modified regexp works, maybe it's an acceptable solution? And if e.g. teleportation scrolls are affected as well, maybe just update those regexps as well.

Otherwise, I don't really know either.

Reply:
This affects every item in that one specific slot. Some regexes work(ish), some don't match at all.